packages/mds-transactions-service/@types/index.ts

```diff
@@ -91,6 +99,9 @@ export interface TransactionSearchParams {
   provider_id?: UUID
   start_timestamp?: Timestamp
   end_timestamp?: Timestamp
+  before?: string
```

Comment:
is before and after intended to be for a timestamp, or an id?

Reply:
I think it's something like `id:1` Base 64 encoded, but that's an implementation detail. It should be treated as an opaque string.